invenio_rdm_records/config.py

```diff
@@ -338,6 +338,24 @@ def make_doi(prefix, record):
 in DataCite XML format.
 """
 
+RDM_CUSTOM_FIELDS_NAMESPACES = {}
+"""Custom fields namespaces.
+
+.. code-block:: python
+
+    {<namespace>: <uri>, ...}
+
+For example:
+
+.. code-block:: python
+
+    {
+        "cern": "https://cern.ch/terms",
```

Comment:
comment: just URI for now, I guess in the future we might require more information.